subcommands/devices/cmd.go

```diff
@@ -60,10 +62,28 @@ func NewCommand() *cobra.Command {
 
 	cmd.AddCommand(configCmd)
 	cmd.AddCommand(updatesCmd)
+
+	addUuidFlagToChildren(cmd)
+
 	return cmd
 }
 
-func getDeviceApi(_ *cobra.Command, name string) client.DeviceApi {
+func addUuidFlagToChildren(c *cobra.Command) {
+	ignores := []string{"list-denied", "list", "delete-denied"}
+	for _, child := range c.Commands() {
+		if child.HasSubCommands() {
+			addUuidFlagToChildren(child)
+		} else if !slices.Contains(ignores, child.Name()) {
+			child.Flags().BoolP("by-uuid", "u", false, "Look up device by UUID rather than name")
+		}
+	}
```

Comment:
This is somewhat implicit, but it looks smart.

Sometimes I look at commands declaration to get the thing about its flags.
That is convenient when I have no access to terminal with Fioctl help and need to quickly answer some question using just GitHub pages.
This change kind of takes that possibility away.

But I think we may make an exception for this use case, as it it rather special.
...As long as we agree to _keep it special and not copy this pattern to other places_.

Reply:
i agree. I hope we don't have to do this again.